test/inductor/test_codegen_triton.py

```diff
@@ -0,0 +1,77 @@
+# Owner(s): ["module: inductor"]
+import contextlib
+
+import sympy
+
+import torch
+
+import torch._inductor.config as inductor_config
+from torch._inductor.codegen import triton_utils
+from torch._inductor.codegen.common import SizeArg
+from torch._inductor.graph import GraphLowering
+from torch._inductor.virtualized import V
+
+from torch.testing._internal.common_utils import TestCase as TorchTestCase
+from torch.testing._internal.inductor_utils import HAS_CPU, HAS_CUDA
+
+
+class TestCodegenTriton(TorchTestCase):
+    def setUp(self):
+        super().setUp()
+
+        class DummyModule(torch.nn.Module):
+            def forward(self, x):
+                return x * 2
+
+        self._gm = torch.fx.symbolic_trace(DummyModule())
+        self._graph = GraphLowering(self._gm)
+
+        self._stack = contextlib.ExitStack()
+        self._stack.enter_context(V.set_graph_handler(self._graph))
+
+    def tearDown(self):
+        self._stack.close()
+        super().tearDown()
```

Comment:
Curious if we need this dummy module setup / teardown for this test.

Reply:
I think we need it because V.graph is used inside triton_utils.config_of. A search in codegen/triton.py also shows V.graph 80 times, so I suspect if we added more unit tests for codegen many of those tests would also benefit from this setup/teardown.